test/publishToNpm.test.js

```diff
@@ -155,3 +155,69 @@ tap.test('Should stop action if package info retrieval fails', async t => {
   ])
   t.pass('package is not published without otp code')
 })
+
+tap.test(
+  'Should stop action if package version info retrieval fails',
```

Comment:
this test has the same name as the previous one

Reply:
the names are currently `package info retrieval` (`npm view --json`) and `package version info retrieval` (`npm view packagename@version`)